.rubocop.yml

```diff
@@ -14,7 +14,7 @@ Metrics/LineLength:
 
 Metrics/MethodLength:
   CountComments: false
-  Max: 26
+  Max: 50
```

Comment:
If it doesn't make sense to split them, this is fine. I'm not aware of any way to filter out logging lines.